pitopcli/display.py

```diff
@@ -0,0 +1,141 @@
+#!/usr/bin/env python3
+from argparse import ArgumentParser
+
+from pitop.display import Display
+from .cli_base import CliBaseClass, PitopCliException
+
+
+class DisplayCLI(CliBaseClass):
+    parser_help = 'communicate and control the device\'s display'
+    cli_name = "display"
+
+    def __init__(self, args) -> None:
+        self.args = args
+        self.validate_args()
+
+    def run(self) -> int:
+        try:
+            self.perform_desired_action()
+            return 0
+        except Exception as e:
+            print(f"Error on pitop-brightness.run: {e}")
+            return 1
+
+    def debug_print(self, text, required_verbosity_lvl=1) -> None:
+        if self.args.verbose is not None and self.args.verbose >= required_verbosity_lvl:
+            print(text)
+
+    def validate_args(self) -> None:
+        # Handle invalid command line parameter combinations
+        if self.args.display_subcommand != "brightness":
+            return
+        if self.args.brightness_value and (self.args.increment_brightness or self.args.decrement_brightness):
+            print("Error on pitop-brightness.validate_args: Cannot increment/decrement at the same time as setting brightness value")
+            raise PitopCliException
+        if self.args.increment_brightness and self.args.decrement_brightness:
+            print("Error on pitop-brightness.validate_args: Cannot increment and decrement brightness at the same time")
+            raise PitopCliException
+
+    def perform_desired_action(self) -> None:
+        display = Display()
+
+        if self.args.display_subcommand == "brightness":
+            if self.args.increment_brightness:
+                self.debug_print("REQ:\tINCREMENTING BRIGHTNESS", 1)
```

Comment:
I feel like we need to have a unified verbosity schema across the CLI. I can't see this actual implementation being of much use to most users, but the general concept of tracing the actions of the CLI would be useful. Perhaps in another PR, we should explore the option of multiple verbosity levels:
```
-v
-vv
-vvv
```